tests/baselines/reference/typeOfThis.errors.txt

```diff
@@ -119,6 +145,8 @@ tests/cases/conformance/expressions/thisKeyword/typeOfThis.ts(130,16): error TS1
 !!! error TS1056: Accessors are only available when targeting ECMAScript 5 and higher.
             var p = this;
             var p: MyGenericTestClass<T, U>;
+                ~
+!!! error TS2403: Subsequent variable declarations must have the same type.  Variable 'p' must be of type 'this', but here has type 'MyGenericTestClass<T, U>'.
             p = v;

```

Comment:
No, they should fail. The first declaration infers type `this` for `v`, the second declaration specifies type `MyGenericTestClass<T, U>` for `v`. Those are not identical types, so error.